### PR TITLE
Add below-suggested-topics plugin outlet

### PR DESCRIPTION
**Pull request by christian-dinh**
Hi! My name is Chris and I'm on the Reach team at Codecademy. I recently added a new Discourse theme component (`Codecademy Related Content`) to all of the user-selectable themes. The component renders correctly on the base Codecademy theme as below:

Screen Shot 2022-02-17 at 12 28 56 PM

but doesn't show up on `Community Rebrand` or `Original Community`, as the `alex_base` component overrides Discourse's default `suggested-topics` component, which normally renders the plugin outlet `Codecademy Related Content` uses.

Long story short, here's the fix! The change can be previewed live via the `Codecademy Community Theme Fork Staging` theme preview in Discourse. Let me know if you have any questions, and feel free to reach out to anyone else at the company to validate this change!

Screen Shot 2022-02-17 at 1 33 23 PM

### CODE REVIEW

**Comment by A-J-C** [comment]
Hi @christian-dinh that's great thanks for doing the work here, the new plugin looks really cool too!
